server/librarian_server/file.py

```diff
@@ -452,6 +452,14 @@ def descriptive_name(self):
     def deletion_policy_text(self):
         return DeletionPolicy.textualize(self.deletion_policy)
 
+    def to_dict(self):
+        return dict(
+            store=self.store,
+            parent_dirs=self.parent_dirs,
+            name=self.name,
+            deletion_policy=self.deletion_policy,
+            full_path_on_store= os.path.join(self.store_object.path_prefix, self.parent_dirs, self.name)
```

Comment:
Stray space after the equals sign here. More importantly, this should call `self.full_path_on_store()`.